src/nvim/api/private/helpers.c

```diff
@@ -808,6 +811,17 @@ static void init_type_metadata(Dictionary *metadata)
 
   PUT(*metadata, "types", DICTIONARY_OBJ(types));
 }
+static void init_api_level_metadata(Dictionary *metadata)
+{
+  Dictionary version = ARRAY_DICT_INIT;
+  PUT(version, "current", INTEGER_OBJ(NVIM_API_CURRENT));
+  PUT(version, "compatibility", INTEGER_OBJ(NVIM_API_COMPATIBILITY));
+#ifdef NVIM_VERSION_PRERELEASE
+  PUT(version, "prerelease", BOOLEAN_OBJ(true));
```

Comment:
wouldn't it be better if `prerelease` was always defined and "false" when not a prerelease. Also I think there will be slight issue: after the 0.1.6 release is done, and in the next commit version is updated to 0.1.7-dev, then `prerelease` will become true even if no api change is done (and thus the level updated). We could have `NVIM_API_PRERELEASE`, and then have a static assert against `NVIM_API_PRERELEASE && !NVIM_VERSION_PRERELEASE`